doc/releases/release-notes-3.5.rst

```diff
@@ -896,6 +896,29 @@ Libraries / Subsystems
 
   * Added support for CAN FD.
 
+* ZBus
+
+  * Changed channels' and observers' metadata to comply with the data/config approach. ZBus stores
+    immutable config in iterable sections in Flash and the mutable portion of data in the RAM.
+  * The relationship between channels and observers is mapped using a new entity called
+    observation. The observation enables us to increase the granularity of masking observation.
+    Developers can mask individual observations, disable the observer, or use runtime observers.
+  * Added API :c:macro:`ZBUS_CHAN_ADD_OBS` macro for adding post-definition static observers of a
+    channel. That can replace the runtime observer feature, enabling developers to add static
+    observers after the channel definition in different files. It increases the composability of
+    the system using ZBus, making post-definition channel observation rely on the stack instead of
+    the heap.
+  * Added a new type of observer called Message Subscriber. ZBus' VDED will send a copy of the
+    message during the publication/notification process.
+  * Changed the VDED delivery sequence. Check the ref:`documentation<zbus delivery sequence>`.
+  * ZBus runtime observers now rely on the heap instead of a memory pool.
+  * Added new iterable section iterators APIs (for channels and observers) can now receive a
+    ``user_data`` pointer to keep context between the function calls.
+  * Added APIs :c:macro:`ZBUS_LISTENER_DEFINE_WITH_ENABLE` and
+    :c:macro:`ZBUS_SUBSCRIBER_DEFINE_WITH_ENABLE` that enable developers to define disabled
+    observers. They need to be enabled during runtime to receive notifications from the bus.
```

Comment:
fwiw this, to me, kind of confirms that the naming is not ideal.
"Use XXX_WITH_**ENABLE** to define a **disabled** observer" is really odd.

Reply:
Do you think there is still time to change that?

Reply:
The description could be better. The API intends to define the "enableness" of an observer programmatically.

Reply:
Yes, better now, thank you!